client/components/App.test.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import App from './App.jsx';
+import ReviewList from './ReviewList.jsx';
+import ReviewItem from './ReviewItem.jsx';
+
+
+
+
+describe('<App />', () => {
+  test('It renders a review list', () => {
+
+    const wrapper = shallow(<App />);
+
+    expect(wrapper.text()).toEqual('<ReviewList />');
+    expect(wrapper.state().ratings).toEqual(0);
+
+  });
+});
+
+
+
+describe('<ReviewList />', () => {
+  test('It should render review list and its props ', () => {
+    //create a comments prop so it has something to map over and an id so it doesnt throw errors
+    let comments = [{
+      id: 1,
+    }];
+
+    const wrapper = shallow(<ReviewList reviews={200} comments={comments} />);
+
+    expect(wrapper.find('.reviewsNumber').text()).toBe('Reviews ***** (200)');
+    expect(wrapper.find('#card').length).toBe(1);
+
+  });
+
+  test('It should render review list and its props ', () => {
+    let comments = [{
```

Comment:
this comment obj can be pulled out into its own JSON file for easier readability